src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTaskTest.java

```diff
@@ -69,8 +69,8 @@ void processBlobs_should_not_call_envelopeProcessor_if_failed_to_load_file() thr
         given(blob.getUri()).willReturn(URI.create("file.zip"));
         given(container.getBlockBlobReference("file.zip")).willReturn(cloudBlockBlob);
         given(container.getName()).willReturn("cont");
-        given(envelopeProcessor.getEnvelopeByFileAndContainer("cont", "file.zip"))
-            .willReturn(null);
+        given(envelopeProcessor.getEnvelopeIdByFileAndContainer("cont", "file.zip"))
+            .willReturn(Optional.empty());
```

Comment:
do we need a test with non empty optional too?

Reply:
Not right now. BlobProcessorTaskTest always contained this only method effectively making BlobProcessorTask not unit tested (all code coverage was provided only by integration tests). After refactoring BlobProcessorTask will become unit testable and more methods will be added to BlobProcessorTaskTest.